cmd/aws-cost-exporter/main.go

```diff
@@ -1,44 +1,74 @@
 package main
 
 import (
-	"github.com/st8ed/aws-cost-exporter/pkg/collector"
-	"github.com/st8ed/aws-cost-exporter/pkg/fetcher"
-	"github.com/st8ed/aws-cost-exporter/pkg/processor"
-	"github.com/st8ed/aws-cost-exporter/pkg/state"
-
 	"context"
+	"net/http"
```

Comment:
these changes simply sort the go packages according to convention:
1. stdlib
2. 3rd party libraries
3. libraries from this module